apps/admin/backend/src/app.test.ts

```diff
@@ -266,3 +265,22 @@ test('getSystemSettings returns default system settings when no system settings
   const systemSettingsResult = await apiClient.getSystemSettings();
   expect(systemSettingsResult).toEqual(DEFAULT_SYSTEM_SETTINGS);
 });
+
+test('saveBallotPackageToUsb', async () => {
+  const { apiClient, auth, mockUsb } = buildTestEnvironment();
+  const { electionDefinition } = electionMinimalExhaustiveSampleFixtures;
+  await configureMachine(apiClient, auth, electionDefinition);
+
+  mockUsb.insertUsbDrive({});
+  const response = await apiClient.saveBallotPackageToUsb();
+  expect(response).toEqual(ok());
```

Comment:
I may come back and add more detailed assertions here but for now am just prioritizing moving quickly so that we can create images for testing this week